app/renderer/js/main.js

```diff
@@ -159,6 +160,31 @@ class ServerManagerView {
 		this.toggleSidebar(showSidebar);
 	}
 
+	async initPresetOrgs() {
+		const presetOrgs = EnterpriseUtil.getConfigItem('presetOrganizations', []);
+		// set to true if at least one new domain is added
+		let domainAdded = false;
+		for (const url in presetOrgs) {
+			if (DomainUtil.duplicateDomain(presetOrgs[url])) {
+				continue;
+			}
+			try {
+				// eslint-disable-next-line no-await-in-loop
+				const serverConf = await DomainUtil.checkDomain(presetOrgs[url], {override: true});
```

Comment:
Using `await` in for loop, as the linting rule enforces, is bad idea. Essentially you are waiting for one operation to finish while other could have been in progress. You should split the code inside of the try block into a async function, then on each loop call the function which will return a promise and store it in a array. Then after the loop do `await Promise.all(<array-of-promise>);`.

Reply:
You're right, I should've taken care of that. 
I'll resolve with `false` when there's an error and with `true` when the domain gets added successfully, since we want to reload if at least one org gets added.